src/components/Main.vue

```diff
@@ -143,22 +143,41 @@ export default {
                 }
             ];
 
+            let firstFlag = issues[0].flags;
+            if(firstFlag.length > 0 && firstFlag[0][0] === 0){
+                flags = [];
+            }
+
             issues.map(issue => 
                 flags = flags.concat(this.mapFlag(issue)));
             return flags;
         },
 
-        getBlurbs(text, flags) {
-
+         getBlurbs(text, flags) {
             //splits and demarcates text into blurbs
-            //using the start and end indexes of flags
-
+            //using the start and end indices of flags
             let textArray = text.split("");
+            // Create an empty element at the beginning of the text so, 
+            // textArray[0] is an empty element instead of the beginning of the text.
+            textArray.unshift(""); 
+            // Modifies the text into flags with their relevance
```

Comment:
What is "relevance"? Do you mean the bias or valence? Do you mean the relevant issue number?